src/cdk/scrolling/virtual-for-of.ts

```diff
@@ -166,14 +167,16 @@ export class CdkVirtualForOf<T> implements CollectionViewer, DoCheck, OnDestroy
       /** The set of available differs. */
       private _differs: IterableDiffers,
       /** The virtual scrolling viewport that these items are being rendered in. */
-      @SkipSelf() private _viewport: CdkVirtualScrollViewport) {
+      @SkipSelf() private _viewport: CdkVirtualScrollViewport,
+      /** The Angular zone. */
+      ngZone: NgZone) {
```

Comment:
Should this be optional with a breaking change for next major?

Reply:
virtual scroll isn't released yet so its ok